pkg/controller/chaosengine/chaosengine_controller.go

```diff
@@ -107,18 +107,18 @@ type applicationInfo struct {
 	experimentList []litmuschaosv1alpha1.ExperimentList
 }
 
-var labelKey string
-var labelValue string
-
-func (a *applicationInfo) Initialize(instance *litmuschaosv1alpha1.ChaosEngine) *applicationInfo {
-	aLabelKeyValue := strings.Split(instance.Spec.Appinfo.Applabel, "=")
-	labelKey = aLabelKeyValue[0]
-	labelValue = aLabelKeyValue[1]
-	a.label = make(map[string]string)
-	a.label[labelKey] = labelValue
-	a.namespace = instance.Spec.Appinfo.Appns
-	a.experimentList = instance.Spec.Experiments
-	return a
+var appLabelKey string
+var appLabelValue string
+
+func (appInfo *applicationInfo) InitializeApplicationInfo(instance *litmuschaosv1alpha1.ChaosEngine) *applicationInfo {
```

Comment:
If we are not using this function outside of the package then it's name should be start from small case